src/BlockTypes/AbstractProductGrid.php

```diff
@@ -562,7 +562,7 @@ protected function get_rating_html( $product ) {
 
 		if ( $rating_count > 0 ) {
 			return sprintf(
-				'<div class="wc-block-grid__product-rating">%s</div>',
+				'<div class="wc-block-grid__product-rating wc-block-components-product-rating">%s</div>',
```

Comment:
I'm not sure about this change... :thinking: IMO Product rating global styles should only change the styles of the Product rating block, but they shouldn't impact PHP product grid blocks because they don't have inner blocks.

Reply:
Ok! I will revert this change